app/components/support_interface/ucas_match_action_component.rb

```diff
@@ -0,0 +1,91 @@
+module SupportInterface
+  class UCASMatchActionComponent < ViewComponent::Base
+    include ViewHelper
+
+    ACTIONS = {
+      send_initial_emails: {
+        description: 'Send initial emails',
+        button_text: 'Confirm initial emails were sent',
+        form_path: :support_interface_record_initial_emails_sent_path,
+      },
+      send_reminder_emails: {
+        description: 'Send reminder emails',
+        button_text: 'Confirm reminder emails were sent',
+        form_path: :support_interface_record_reminder_emails_sent_path,
+      },
+      request_withdrawal_from_ucas: {
+        description: 'Request withdrawal from UCAS',
+        button_text: 'Confirm withdrawal from UCAS was requested',
+        form_path: :support_interface_record_ucas_withdrawal_requested_path,
+        instructions: "We need to contact UCAS. Please send an encrypted file with the candidate's duplicate application details to Harry Haines (h.haines@ucas.ac.uk) and Lizzy Carter (l.carter@ucas.ac.uk) from UCAS to ask them to remove the candidate from UTT.",
+      },
+      confirm_withdrawal_from_ucas: {
+        description: 'Confirm withdrawal from UCAS',
+        button_text: 'Confirm the application was withdrawn from UCAS',
+        form_path: :support_interface_process_match_path,
+        instructions: "We need to ensure that UCAS have removed the candidate's duplicate application from UTT by the end of the day. Continue to contact UCAS until this has been confirmed.",
```

Comment:
Does "by the end of the day" make sense here — what if they clicked the above button late in the afternoon?

Reply:
This is the copy I got from Natasha 🤷‍♀️  She said that UCAS should remove them immediately.